artemis-features/src/main/resources/artemis.xml

```diff
@@ -196,5 +194,20 @@ under the License.
 
       </addresses>
 
+
+      <!-- Uncomment the following if you want to use the Standard LoggingActiveMQServerPlugin pluging to log in events
```

Comment:
Why add this?

Reply:
As mentioned before it's coming from default.